src/wp-includes/update.php

```diff
@@ -72,7 +72,7 @@ function wp_version_check( $extra_stats = array(), $force_check = false ) {
 
 	// Update last_checked for current to prevent multiple blocking requests if request hangs.
 	$current->last_checked = time();
-	set_site_transient( 'update_core', $current );
+	set_site_transient_nohook( 'update_plugins', $current );
```

Comment:
Why it change `update_core` to `update_plugins`?

Reply:
so that is runs only on plugin update not core update
it was in the original patch